docs/src/pages/docs/Creating Charts and Dashboards/exploring-data.mdx

```diff
@@ -109,40 +101,42 @@ dashboard** and then hover over the table. By selecting the bottom right hand co
 
 Finally, save your changes by selecting Save changes in the top right.
 
-### Pivot Table
+### Pivot Table v2
 
-In this section, we will extend our analysis using a more complex visualization, Pivot Table. By the
+In this section, we will extend our analysis using a more complex visualization, Pivot Table v2. By the
 end of this section, you will have created a table that shows the monthly spend on flights for the
 first six months, by department, by travel class.
 
 Create a new chart by selecting **+ ‣ Chart** from the top right corner. Choose
 tutorial_flights again as a datasource, then click on the visualization type to get to the
-visualization menu. Select the **Pivot Table** visualization (you can filter by entering text in the
+visualization menu. Select the **Pivot Table v2** visualization (you can filter by entering text in the
 search box) and then **Create New Chart**.
 
-<img src="/images/create_pivot.png" />
```

Comment:
can we remove create_pivot.png and all the other old pictures?

Reply:
done!